node/Cargo.toml

```diff
@@ -0,0 +1,182 @@
+[badges]
+travis-ci = { repository = "centrifuge/centrifuge-chain", branch = "master" }
+maintenance = { status = "actively-developed" }
+is-it-maintained-issue-resolution = { repository = "centrifuge/centrifuge-chain" }
+is-it-maintained-open-issues = { repository = "centrifuge/centrifuge-chain" }
+
+[package]
+name = "centrifuge-chain"
+version = "0.10.34"
+description = "Centrifuge chain implementation in Rust."
+build = "build.rs"
+default-run = "centrifuge-chain"
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+documentation.workspace = true
+
+[package.metadata.wasm-pack.profile.release]
+# `wasm-opt` has some problems on linux, see
+# https://github.com/rustwasm/wasm-pack/issues/781 etc.
+wasm-opt = false
```

Comment:
Does anyone know what was exactly the issue to check if it now works removing this? From the linked issue, it seems fixed, but I am not sure how to check it or evaluate it